Build of LISA with plugins breaks with "Cannot find module: '@apollo/react-hooks'"

I just noticed that the build of LISA with a plugins fails due to:
```sh
$ npm run build
> lisa@0.1.0 build /builds/lisa
> react-scripts build
Creating an optimized production build...
Failed to compile.
/builds/lisa-plugin/dist/index.js
Cannot find module: '@apollo/react-hooks'. Make sure this package is installed.
You can install this package by running: npm install @apollo/react-hooks.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! lisa@0.1.0 build: `react-scripts build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the lisa@0.1.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     /home/node/.npm/_logs/2022-04-20T14_32_43_821Z-debug.log
```

--- a/lisa-example-plugin/src/components/ModelSelectInput.js
+++ b/lisa-example-plugin/src/components/ModelSelectInput.js
@@ -1,6 +1,6 @@
 import React, { useContext} from 'react'
 import T from 'i18n-react'
-import { useQuery } from '@apollo/react-hooks'
+import { useQuery } from '@apollo/client'
 import { get } from 'lodash'
 import { HostsFormContext } from 'lisa-core/lib/Context'
 import SelectInput from 'lisa-core/components/HostsForm/SelectInput'

